bihire/Broadcatser
# #169626169 #169680805 #169680677 #169668646 admin page, user profile page, edit page, specific red-flag/intervention page

### What does this Pr do ?
admin page, 
user profile page, 
edit page, 
specific red-flag/intervention page
### Description of the task to be completed ?
- admin page where admin can see all red-flags/interventions created by all users
- profile page for an individual user with cards indicating all red-flags/interventions that are under-investigation, resolved or rejected
- see a specific red-flag/intervention where admin can change status of the flag or user can edit the flag
- a page where user can edit their own red-flags/interventions
- all the pages should be responsive

### How should this be manually tested?
after cloning this repository
- dive into the directory named UI you should be able to see all the pages above, open them using chrome you should be able to see all the changes named above
### Any background context you want to provide?
N/A
### What are the relevant pivot tracker stories ?
#16962616
 #169680805
#169680677
#169668646
https://www.pivotaltracker.com/story/show/169680805
https://www.pivotaltracker.com/story/show/169680677
https://www.pivotaltracker.com/story/show/169668646
https://www.pivotaltracker.com/story/show/169626169
#### screenshots if available?
![image](https://user-images.githubusercontent.com/37307172/68598560-bae76d80-04a7-11ea-8975-9c7f2b9ba88d.png)
![image](https://user-images.githubusercontent.com/37307172/68598623-d2265b00-04a7-11ea-8cff-dc71596ae584.png)
![image](https://user-images.githubusercontent.com/37307172/68598662-e10d0d80-04a7-11ea-9c0c-c473953db59b.png)
![image](https://user-images.githubusercontent.com/37307172/68598702-f5e9a100-04a7-11ea-8cb8-36c708bc52d9.png)
